spec/tapioca/cli_spec.rb

```diff
@@ -602,6 +602,34 @@ class Foo::Secret
       assert_path_exists("#{outdir}/baz@0.0.2.rbi")
     end
 
+    it 'generate output  Done (empty output) after generating an empty RBI file' do
+      execute("generate", ["foo"])
+
+      output = execute("sync")
+
+      assert_includes(output, "++ Adding: #{outdir}/ast@2.4.1.rbi\n")
+      assert_includes(output, <<~OUTPUT)
+        Compiling ast, this may take a few seconds...   Done (empty output)
+      OUTPUT
+    end
+
+    it 'Validate the content of a generated empty RBI file' do
+      execute("generate", ["foo"])
+      execute("sync")
+
+      assert_equal(<<~CONTENTS.chomp, File.read("#{outdir}/ast@2.4.1.rbi"))
```

Comment:
Actually, I feel like this is just one more assertion for the single scenario outlined above, so we should just merge the two tests into one. Basically, we are setting up a single scenario where we know there is an empty RBI that is being generated and we are trying to assert that all the expectations as a result of that happening are behaving as expected.

Thus, there is no need to setup the scenario again in a separate test.

Reply:
Wouldn't tests like these would have to be updated when `ast` releases a version `2.4.2`?

Previously this file never relied on real gems. It only uses `bar`, `foo`, `baz` . I think we can achieve the same functionality without checking for "existence of" or "contents of" `ast@2.4.1.rbi`

Reply:
I am going to add a fake gem and use that one. :)